Docs: add mimir recommended scrape interval (#2147)
grafana/mimir: docs/sources/operators-guide/visualizing-metrics/requirements.md

| `pod`       | Yes          | The unique identifier of a Mimir replica (eg. Pod ID when running on Kubernetes). The label name can be configured with the `per_instance_label` field in the mixin config.                                                                                                                                                                                                                                                                                                                      |
| `instance`  | Yes          | The unique identifier of the node or machine where the Mimir replica is running (eg. the node when running on Kubernetes). The label name can be configured with the `per_node_label` field in the mixin config.                                                                                                                                                                                                                                                                                 |

> For dashboards and rules to function properly, your Prometheus or Grafana Agent must be configured to scrape metrics from Grafana Mimir at a 10s interval.

Reviewer: Problem is that many [alerts use 1m rates](https://github.com/grafana/mimir/blob/main/operations/mimir-mixin/alerts/alerts.libsonnet), and we should be using [4x of the scrape interval](https://www.robustperception.io/what-range-should-i-use-with-rate/). Since we're using 1m, scrape interval should be max `15s`.

I chose `10s` for the demo.

In dashboards we use `$__rate_interval` variable provided by Grafana, which is based on data-source configuration.

```suggestion
For rules and alerts to function properly, your Prometheus or Grafana Agent must be configured to scrape metrics from Grafana Mimir at a `15s` interval or shorter.
```

> For rules and alerts to function properly, your Prometheus or Grafana Agent must be configured to scrape metrics from Grafana Mimir at a `15s` interval or shorter.

Reviewer 2: ```suggestion
For rules and alerts to function properly, you must configure your Prometheus or Grafana Agent to scrape metrics from Grafana Mimir at an interval of `15s` or shorter.
```
